packages/node_modules/@webex/test-users/src/whistler.js

```diff
@@ -0,0 +1,197 @@
+import btoa from 'btoa';
+import {request} from '@webex/http-core';
+
+/**
+ * Computes `expires` and `refresh_token_expires` from `expires_in` and
+ * `refresh_token_expires_in` and creates an `authorization` string.
+ * @param {Object} token
+ * @private
+ * @returns {Object}
+ */
+function fixToken(token) {
+  const now = Date.now();
+
+  if (token.expires_in && !token.expires) {
+    token.expires = now + token.expires_in * 1000;
+  }
+
+  if (token.refresh_token_expires_in && !token.refresh_token_expires) {
+  /* eslint camelcase: [0] */
+    token.refresh_token_expires = now + token.refresh_token_expires_in * 1000;
+  }
+
+  if (token.token_type && token.access_token) {
+    token.authorization = `${token.token_type} ${token.access_token}`;
+  }
+
+  return token;
+}
+
+let clientToken;
```

Comment:
Why would you want to cache the user? Isn't the expectation that I will get a new user every time I call "create user"?